src/content.ts

```diff
@@ -84,9 +72,24 @@ function listen(elem) {
     )
 }
 listen(window)
-document.addEventListener("readystatechange", _ =>
-    getAllDocumentFrames().forEach(f => listen(f)),
```

Comment:
This registered event handlers on `<iframe>` elements, which are still a part of the same document tree as root. I don't think they ever captured anything.